components/event-feed-service/pkg/persistence/feed-elastic.go

```diff
@@ -134,124 +135,88 @@ func (efs ElasticFeedStore) CreateFeedEntry(entry *util.FeedEntry) (bool, error)
 	return true, nil
 }
 
+// GetFeed - get event feed entries with the provided query constraints
 func (efs ElasticFeedStore) GetFeed(query *util.FeedQuery) ([]*util.FeedEntry, int64, error) {
 	exists, err := efs.indexExists(IndexNameFeeds)
 	if err != nil {
 		logrus.Warnf("Could not get feed; error: %+v", err)
 		return nil, 0, err
 	}
+	if !exists {
```

Comment:
Made this more go like 😄

Reply:
If this is something the service can't recover from, failing fast seems like a good idea 👍